### PR TITLE
feat(pagination): Adding component - FRONT-1781

### PR DESCRIPTION
**Pull request by papegaill**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
- [ ] I have checked the dependencies
- [ ] I have given the fractal status “ready” to my component
- [ ] I have declared `@define mycomponent` in the SCSS file
- [ ] I have specified `margin: 0;` on the CSS component
- [ ] I have provided tests
- [ ] I follow the naming guidelines
- [ ] the component supports composition
- [ ] there are no hardcoded strings (all content come from the context)
- [ ] I have filled the README.md file (at least a few lines)
- [ ] My component is listed in the root README
- [ ] My PR has the right label(s)
- [ ] The redirects in `src/website/src/routes/Redirects.jsx` are up-to-date

### CODE REVIEW

File: src/website/src/pages/ec/components/navigation/pagination/docs/code.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+order: 2
+---
+
+import { Playground, Html } from '@ecl/website-components';
+
+import pagination from '../demo';
+
+<Playground
+  system="ec"
+  selectedKind="components-pagination"
```
Comment (emeryro) [inline src/website/src/pages/ec/components/navigation/pagination/docs/code.mdx]:
```suggestion
  selectedKind="components-navigation-pagination"
```

File: src/implementations/twig/components/pagination/pagination.test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import { merge, renderTwigFileAsNode } from '@ecl/test-utils';
+
+// Import data for tests
+import data from '@ecl/specs-component-pagination/demo/data';
+
+describe('EC - Pagination', () => {
```
Comment (emeryro) [inline src/implementations/twig/components/pagination/pagination.test.js]:
```suggestion
describe('Pagination', () => {
```

File: src/implementations/vanilla/components/pagination/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@ecl/vanilla-component-pagination",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "3.0.0",
+  "description": "ECL Pagination",
+  "style": "pagination.scss",
+  "sass": "button.scss",
```
Comment (emeryro) [inline src/implementations/vanilla/components/pagination/package.json]: bad copy paste I guess

File: src/specs/components/pagination/package.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@ecl/specs-component-pagination",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "3.0.0",
+  "description": "ECL EC Pagination Specs",
```
Comment (emeryro) [inline src/specs/components/pagination/package.json]:
```suggestion
  "description": "ECL Pagination Specs",
```

**Review by emeryro — CHANGES_REQUESTED** [review]
Almost good, just a few points below

**Comment by emeryro** [comment]
@papegaill the tests have to be updated here

File: src/implementations/vanilla/components/pagination/package.json (hunk 3 above)
Comment (planctus) [inline src/implementations/vanilla/components/pagination/package.json]:
```suggestion
  "sass": "pagination.scss",
```

File: src/presets/dev/package.json [hunk 5]
```diff
@@ -29,6 +29,7 @@
     "@ecl/vanilla-component-menu": "^3.0.0",
     "@ecl/vanilla-component-message": "^3.0.0",
     "@ecl/vanilla-component-ordered-list": "^3.0.0",
+    "@ecl/vanilla-component-pagination": "^3.0.0",
```
Comment (planctus) [inline src/presets/dev/package.json:32]: Shouldn't come after page-banner? :)

**Comment by planctus** [comment]
in the website, we miss the arrows of the previous and next link: 
https://ecl-preview-1759--europa-component-library.netlify.app/ec/components/navigation/pagination/code/

image

**Review by planctus — APPROVED** [review]

**Pull request merged**